tests/perftest_make.sh

```diff
@@ -29,10 +29,13 @@ fi
 
 # Don't overwrite existing data; if we want to compare multiple
 # binaries then we should rename the output file.
-if [ -e "${data}" ]; then
-	printf "Error: ${data} already exists; will not overwrite\n"
-	exit 1
-fi
+#if [ -e "${data}" ]; then
+#	printf "Error: ${data} already exists; will not overwrite\n"
+#	exit 1
+#fi
+
+# Delete previous data.
+rm -f "${data}"
```

Comment:
I can't decide whether this should
1) carefully avoid overwriting any old data, or
2) cheerfully delete old data

I originally went with 1, but on second thought it takes less than a minute to recompile and re-run any of our perftests, so now I'm leaning towards 2.

Reply:
I'm ok with deleting/overwriting old test results.  In fact, don't we do that for the regular tests anyway?  (For tests where the C code generates a file and then we compare it to "known good" output?)

Reply:
Yeah, but deleting output that takes 0.2 seconds to generate is a no-brainer compared to deleting output that takes a minute to generate.

Ok, going with delete.